### DISCUSSION TITLE
I have created a password input with a show and hide password button

### DISCUSSION THREAD

**Discussion by diegosyswork** [vikramlearning/blazorbootstrap#948] [Ideas]
![image](https://github.com/user-attachments/assets/26676754-cc50-45ea-b576-071d6163ab56)

![image](https://github.com/user-attachments/assets/01120a92-301e-4404-8e1c-c8363dbcc139)

I would like to share the code with the creator of Blazorbootstrap, so that the control is part of the framework. Does anyone know how to contact him?

**Comment by gvreddy04** [comment]
@diegosyswork Thank you for using BlazorBootstrap and for creating the new component. Please create a PR with the new component or share the GitHub repo. I'll take care of adding it.

**Reply by diegosyswork** [reply]
Hi @gvreddy04, I have forked your repository and added the password input control and tried to add a demo, but it didn't work locally. You can check it out.
I am interested in collaborating with the project, I am thinking of making an rdlc report viewer, using the PDFViewer that you developed. I will keep you posted on the news and then share it with you.

https://github.com/diegosyswork/blazorbootstrap
